todo_app/app.py

```diff
@@ -9,24 +10,38 @@
 
 @app.route('/')
 def index():
-    items = sorted(get_items(), key=lambda item: 1 if item['status'] == "Not Started" else 2)
-    return render_template('index.html', items=items)
+    items = get_items()
+    not_started_items = [item for item in items if item.status=="Not Started"]
+    complete_items = [item for item in items if item.status=="Complete"]
+    return render_template('index.html', not_started_items=not_started_items, complete_items=complete_items)
 
 @app.route('/add', methods=["POST"])
 def add():
-    new_item = request.form.get('item_title')
-    add_item(new_item)
+    new_item_title = request.form.get('item_title')
+    add_item(new_item_title)
     return redirect('/')
 
-@app.route('/toggle', methods=["POST"])
-def toggle_complete():
-    item_to_toggle = get_item(request.form.get('itemId'))
-    item_to_toggle['status'] = 'Complete' if item_to_toggle['status'] == 'Not Started' else 'Not Started'
-    save_item(item_to_toggle)
+@app.route('/complete', methods=["POST"])
+def complete():
+    item_to_mark_complete = get_item(request.form.get('itemId'))
+    item_to_mark_complete.status = 'Complete' 
+    save_item(item_to_mark_complete)
+    return redirect('/')
+
+@app.route('/uncomplete', methods=["POST"])
+def uncomplete():
+    item_to_mark_to_do = get_item(request.form.get('itemId'))
+    item_to_mark_to_do.status = 'Not Started'
+    save_item(item_to_mark_to_do)
     return redirect('/')
 
 @app.route('/remove', methods=['POST'])
 def remove():
     itemId = request.form.get('itemId')
     remove_item(itemId)
     return redirect('/')
+
+@app.errorhandler(TrelloApiError)
+def handle_trello_api_error(e):
+    print(e)
+    return redirect('/')
```

Comment:
Nice addition - worth bearing in mind that an error from the Trello API on your index route will infinitely loop here though!